source/compatibility.tex

```diff
@@ -2155,7 +2156,7 @@
 In \Cpp{}, class members can be declared with the \tcode{static} storage
 class specifier.
 Allowing storage class specifiers on type
-declarations could render the code confusing for users.
+declarations can render the code confusing for users.
```

Comment:
_Allowing_ them can't render the code confusing because we don't allow them.  Just strike "Allowing" (implicitly describing C instead).

Reply:
Ditto